apps/client/src/app/pages/portfolio/analysis/analysis-page.html

```diff
@@ -66,6 +66,7 @@ <h1 class="d-none d-sm-block h3 mb-3 text-center" i18n>Analysis</h1>
               />
             </div>
           </div>
+          <div><hr /></div>
```

Comment:
The line stands for a sum and is intentionally not there 😏

Reply:
Aha, sorry, I just assumed it had been forgotten :sweat_smile: